frontend/src/ui/Input.tsx

```diff
@@ -91,24 +93,29 @@ export const CopyableInput: React.FC<CopyableInputProps> = ({
         height: "100%",
         padding: `4px ${buttonSize + 10}px 4px 10px`,
     };
+    const sharedProps = {
+        disabled: true,
+        value: value,
+        "aria-labelledby": context,
+    };
     const inner = multiline
-        ? <textarea disabled value={value} css={{
+        ? <textarea {...sharedProps} css={{
             ...sharedStyle,
             overflow: "hidden",
             resize: "none",
         }} />
-        : <input disabled value={value} css={sharedStyle} />;
+        : <input {...sharedProps} css={sharedStyle} />;
 
     return (
-        <div css={{
+        <div id={context} css={{
```

Comment:
Using this as ID (and especially with my previous suggestions) is not a great idea I think. Rather generate a random ID in these cases where an actual `id` is required. You can use [`useId`](https://react.dev/reference/react/useId) for that.